runtime/bundles/org.eclipse.core.jobs/src/org/eclipse/core/runtime/jobs/IJobChangeListener.java

```diff
@@ -119,4 +126,22 @@ public interface IJobChangeListener {
 	 * @param event the event details
 	 */
 	void sleeping(IJobChangeEvent event);
+
+	/**
+	 * Static helper method to create an <code>IJobChangeListener</code> for the
+	 * {@link #done(IJobChangeEvent)}) method, given a lambda expression or a method
+	 * reference.
+	 *
+	 * @param c the consumer of the event
+	 * @return IJobChangeAdapter
+	 * @since 3.14
+	 */
+	static IJobChangeListener jobDoneAdapter(Consumer<IJobChangeEvent> c) {
```

Comment:
`onDone` as a method name would be enough, I think there is no purpose in exposing the internal implementation in the method name.

Reply:
done